src/main/java/org/prebid/server/geolocation/MaxMindGeoLocationService.java

```diff
@@ -0,0 +1,77 @@
+package org.prebid.server.geolocation;
+
+import com.maxmind.db.Reader;
+import com.maxmind.geoip2.DatabaseReader;
+import com.maxmind.geoip2.exception.GeoIp2Exception;
+import io.vertx.core.Future;
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveInputStream;
+import org.prebid.server.exception.PreBidException;
+import org.prebid.server.execution.Timeout;
+import org.prebid.server.geolocation.model.GeoInfo;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.InetAddress;
+import java.util.Objects;
+import java.util.zip.GZIPInputStream;
+
+/**
+ * Implementation of <a href="https://dev.maxmind.com/geoip/geoip2/geolite2/">MaxMind free database</a>
```

Comment:
That's actually implementation of `GeoLocationService` backed by MaxMind free database.